test/SparqlAntlrParserTestHelpers.h

```diff
@@ -335,6 +335,16 @@ auto VariableOrderKey =
       AD_FIELD(::VariableOrderKey, isDescending_, testing::Eq(desc)));
 };
 
+auto VariableOrderKeys =
+    [](const std::vector<std::pair<std::string, bool>>& orderKeys)
+    -> testing::Matcher<const std::vector<::VariableOrderKey>&> {
```

Comment:
I think inside at the beginning of the current namespace (is it `matchers`? ) you can put a 
`using testing::Matcher`.

and then always write `Matcher<...>` istead of `testing::Matcher<...>` which increases readability.